### PR TITLE
Variety Update

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Adds a variety of features and bugfixes which will be mentioned in the changelog.

Screenshots of some features because I am lazy (kinda outdated).
https://cdn.discordapp.com/attachments/500801682058379265/756410553123078235/unknown.png
https://cdn.discordapp.com/attachments/500801682058379265/756411664060121148/unknown.png
https://cdn.discordapp.com/attachments/500801682058379265/756411417112084550/unknown.png

## Why It's Good For The Game
Because.

## Changelog
:cl:
add: Added more information about the USL's language strings (both elite and normal.)
add: Now USL (and moths) will speak in their language by default, use :0 to speak in Galactic Common again.
add: Marine shoes now come with knives available at GHMME vendors (with the help of Tivi).
add: Added the intro text for both survivor and AI.
add: Made the "You survived the attack... until now" text support more maps for survivors.
add: Added more dreaming strings.
add: Added a more obvious indicator when you die (that differentiates between species!), now new players are encouraged not to disconnect when they will die (with the help of Hugh).
del: Removed a legacy text about Cargo techs.
tweak: Enlarged the ERT role text (You are a... [role]!).
tweak: Enhanced the Language menu.
tweak: Talking to the ERT radio with ; now goes to whatever ERT channel is meant for (bugged, somehow the frequency is for the Alamo instead)
fix: Fixed both SoM and (unused) Echo headsets.
spellcheck: Removed an uneeded end quotation in "X human\s on the ship."
spellcheck: Added "a / an" to the job intro text (for the AI).
spellcheck: Renamed the ERT headsets.
spellcheck: Other misc. respellings.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
@Hughgent @TheBimmer231 Any uhhh review?

**Comment by TiviPlus** [comment]
Bimmer only does sprite stuff

File: code/modules/mob/living/carbon/xenomorph/death.dm [hunk 1]
```diff
@@ -45,6 +45,8 @@
 	if (isAI)
 		gib()
 
+	to_chat(src,"<b><span class='deadsay'><p style='font-size:1.5em'><big>We have perished.</big><br><small>But it is not the end of us yet... wait until a newborn can rise in this world...</small></p></span></b>")
```
Comment (Hughgent) [inline code/modules/mob/living/carbon/xenomorph/death.dm:48]: My recommendation about this was to add a `special_death_message` into the caste datum for each xeno. But this is also acceptable.

**Review by Hughgent — APPROVED** [review]
LGTM

File: code/datums/jobs/job/job.dm [hunk 2]
```diff
@@ -143,8 +143,8 @@ GLOBAL_PROTECT(exp_specialmap)
 /datum/job/proc/radio_help_message(mob/M)
 	to_chat(M, {"
 <span class='role_body'>|______________________|</span>
-<span class='role_header'>You are a: [title]!</span>
-<span class='role_body'>As a [title] you answer to [supervisors]. Special circumstances may change this.</span>
+<span class='role_header'>You are a / an [title]!</span>
+<span class='role_body'>As a / an <b>[title]</b> you answer to [supervisors]. Special circumstances may change this.</span>
```
Comment (TiviPlus) [inline code/datums/jobs/job/job.dm:147]: would the byond \an  text macro work here?
What it does is as I understand is automatically add a/an as needed

Comment (XSlayer300) [reply code/datums/jobs/job/job.dm:147]: I guess so, but I do not know how to do that.

Comment (TiviPlus) [inline code/datums/jobs/job/job.dm:147]:
```suggestion
<span class='role_header'>You are \an [title]!</span>
<span class='role_body'>As \an <b>[title]</b> you answer to [supervisors]. Special circumstances may change this.</span>
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**